Readme.md

```diff
@@ -21,14 +21,9 @@ You'll need poetry to create the python virtual environment to run this project.
 
 Once you have installed poetry, follow this steps to generate the filtered data.
```

Comment:
Follow "these" steps.

Reply:
When I ran `poetry install`, I got the message: 
```
Warning: The current project could not be installed: No file/folder found for package copa-data-filtering
```
This message went away once I created the folder 'copa-data-filtering'. Are you using this folder?